subsys/bluetooth/controller/ll_sw/ull_llcp_common.c

```diff
@@ -127,10 +131,19 @@ static void lp_comm_ntf(struct ull_cp_conn *conn, struct proc_ctx *ctx)
 	ntf->hdr.type = NODE_RX_TYPE_DC_PDU;
 	pdu = (struct pdu_data *) ntf->pdu;
 
-	switch (ctx->proc) {
-	case PROC_VERSION_EXCHANGE:
+	switch (conn->response_opcode) {
```

Comment:
Not sure i'm in favor of this.. think it's dangerous to tie host notification up on the received pdu and not on the active procedure.

Reply:
Hm, good point, probably we should keep the check for proper procedure, but we'll still have to  check that we received the proper PDU type back before decoding it. I'll think of something

Reply:
Ok, added some better checking here